### DISCUSSION TITLE
gql 방식은 어떨까요

### DISCUSSION THREAD

**Discussion by mxxxxwoo** [changdoc/naver-vaccine-macro#53] [Ideas]
지도페이지에서 gql 요청날려서 백신 갯수 체크해서 넘어가는형식으로

**Comment by changdoc** [comment]
네 지도에서 새로고침을 반복적으로 날려 잔여 백신 수가 확인되는 경우 예약 신청 페이지로 이동 시키는 형태를 고민중입니다.
graphql을 직접 요청하는 방식도 가능할듯하긴한데 지도 페이지에서 동작하는 형태와 동일하게 요청이 진행되어야 차단당하는일이 없을거 같긴하네요.
